package/skelcd-control-SLES4SAP.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Dec 11 17:01:30 UTC 2018 - dgonzalez@suse.com
+
+- Use FHS compliant path (fate#325482)
+- 15.0.6
```

Comment:
Same case than https://github.com/yast/skelcd-control-SLE_HPC/pull/19 We should either use version 15.1.0 or submit this also to the SLE-15-GA branch.